app/helpers/reference_helper.rb

```diff
@@ -0,0 +1,38 @@
+module ReferenceHelper
+
+  def after_desired_date?
+    Time.zone.today > Date.parse('2015-12-31')
```

Comment:
We can set ENVs ourselves, without anyone's involvement. What I think would be good is to read ENV, but fallback to 1.1.2016. So like that, we don't even have to high state production environments when we deploy. We'd just set it on demo and branch builder will automatically pick it up.

Moving time on a server would be more work I think, everybody would need to be ready at the same time and there's risk involved. Also some data would be created in the future and then the time would move back..